vue/components/public/event-emitter-mixin.vue

```diff
@@ -3,28 +3,41 @@ import _ from 'lodash';
 
 export default {
   name: 'EventEmitterMixin',
-  emits: ['value-changed'],
-  data() { return { dfEventEmitter: false }; },
-  computed: {
-    parentEmitter() {
+  methods: {
+    getHandler(functionName, getPayload = false) {
       let parent = this.$parent;
-      while (parent && parent.dfEventEmitter !== true) parent = parent.$parent;
+      let payload = null;
+      while (parent && parent[functionName] === undefined) {
+        parent = parent.$parent;
+        if (parent && parent.payload !== undefined && !payload) {
+          payload = parent.payload;
+        }
+      }
+      if (getPayload) {
```

Comment:
Tukaj imaš malo jebe, ker se zna zgoditi, da imaš handler, ampak do njega še nimaš payloada. Tako ti getpayload ne bo vrnila ničesar. Poleg tega bi predlagal, da funkcija vedno vrne oboje in to razbiješ v dve spremenljivki spodaj v dispatchAction

Reply:
done